src/sidebar/store/modules/frames.js

```diff
@@ -7,6 +7,10 @@ import shallowEqual from 'shallowequal';
 
 import * as util from '../util';
 
+/**
+ * @typedef {import('../../services/frame-sync').Frame} Frame
```

Comment:
Can you move the typedef from `services/frame-sync` to this module to keep the overall dependency structure where modules in `services` depend on modules in `store` but not vice-versa.